Fix broken link

flegaspi700 opened the pull request:
Fix broken "Analyzing" link and uploaded an English version of the README.md file for Communication.

jlooper commented on 4-Data-Science-Lifecycle/15-analyzing/README.md:
```diff
@@ -34,10 +34,10 @@ General querying of the data can help you answer some general questions and theo
 
 ## Exploring with Visualizations
-You don’t have to wait until the data is thoroughly cleaned and analyzed to start creating visualizations. In fact, having a visual representation while exploring can help identify patterns, relationships, and problems in the data. Furthermore, visualizations provide a means of communication with those who are not involved with managing the data and can be an opportunity to share and clarify addi ...
+You don’t have to wait until the data is thoroughly cleaned and analyzed to start creating visualizations. In fact, having a visual representation while exploring can help identify patterns, relationships, and problems in the data. Furthermore, visualizations provide a means of communication with those who are not involved with managing the data and can be an opportunity to share and clarify addi ...
```
hi, rather than the full link, it's preferable to add relative links here

flegaspi700 replied:
The relative links returns a Page not found error. I made some modification, so it doesn't have the full link.

jlooper replied:
can you link to a README.md file using a relative link?

jlooper commented on 4-Data-Science-Lifecycle/15-analyzing/README.md:
```diff
@@ -34,10 +34,10 @@ General querying of the data can help you answer some general questions and theo
 
 ## Exploring to identify inconsistencies
-All the topics in this lesson can help identify missing or inconsistent values, but Pandas provides functions to check for some of these. [isna() or isnull()](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.isna.html) can check for missing values. One important piece of exploring for these values within your data is to explore why they ended up that way in the first place. This  ...
+All the topics in this lesson can help identify missing or inconsistent values, but Pandas provides functions to check for some of these. [isna() or isnull()](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.isna.html) can check for missing values. One important piece of exploring for these values within your data is to explore why they ended up that way in the first place. This  ...
```
same here

flegaspi700 commented:
> hi, let's change these to relative links please

I've updated the links to "/3-Data-Visualization" and "/2-Working-With-Data/08-data-preparation/notebook.ipynb".
I just added "/" for each, let me know if that works for you.

jlooper reviewed (approved):
looks good, thanks

jlooper commented:
@flegaspi700 looks like there's a merge conflict, could you fix please then I will merge to main. thanks!

flegaspi700 commented:
> @flegaspi700 looks like there's a merge conflict, could you fix please then I will merge to main. thanks!

Done.

flegaspi700 commented:
I'll close this pull request and create a new one.
